releases/release-1.21/release_team.md

```diff
@@ -0,0 +1,16 @@
+# Kubernetes 1.21 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
+|----------|----------------------------------|----------------------------------------|
+| Lead | Nabarun Pal ([@palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`) | Kirsten Garrison ( [@kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`), Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) / Slack: `@sraghunathan`), Vlad Gorodetsky ([@bai](https://github.com/bai) / Slack: `@bai`) |
+| Enhancements | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`) | |
+| CI Signal | Joyce Kung ([@thejoycekung](https://github.com/thejoycekung) / Slack: `@jk`) | |
+| Bug Triage | Derrik Campau ([@erismaster](https://github.com/erismaster) / Slack: `@dcampau`) | |
+| Docs | Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rey.lejano`) | |
+| Release Notes | Wilson Husin ([@wilsonehusin](https://github.com/wilsonehusin) / Slack: `@wilsonehusin`) | |
+| Communications | Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) / Slack: `@divya`) | |
+| Emeritus Adviser | Taylor Dolezal ([@onlydole](https://github.com/onlydole) / Slack: `@onlydole`) | |
+
+Review the [Release Managers page](/release-managers.md) for up-to-date contact information on Release Engineering personnel.
+
+The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.20, once the 1.20 release cycle concludes.
```

Comment:
```suggestion
The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.21, once the 1.21 release cycle concludes.
```